src/docs/GetStarted/Developers/CustomizeTokens/index.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Tokens customization'
+order: 9
+---
+
+# Principles
+
+Under the hood, mozaic SCSS uses the compiled tokens that we provide in the `@mozaic-ds/tokens` npm package. This dependency is imported via `@mozaic-ds/styles` by default.
+
+Mozaic tokens are based upon the LeroyMerlin brand guidelines but you can adapt thoses tokens to fit your brand identity.
+
+**It is possible for you to override any value, as well as extend them to create new things**.
+
+## About tokens :
+
+As a reminder, we call tokens a series of Json files containing mostly color values (ammong others things). Thoses files are transpiled into SCSS, js, ios, and android formats, to be used within multiple platforms and technologies.
+
+for more informations, read the following documentation :
+
+[Getting started : Variables and design tokens](/GetStarted/Developers/CssNamingConventions/#variables-and-design-tokens)
+
+[Contributting : design tokens](/Contributing/Prerequisite/DesignTokens/)
+
+## 1 - Configure mozaic
+
+If it is not already done, create a `mozaic.config.js` file at your project root. Then define those keys :
+
+```js
+module.exports = {
+  tokens: {
+    localTokensSrcPath: './src/tokens/', // path to your own js files
+    localTokensExportPath: './dest/build/', // path to compiled files
+  },
+}
+```
+
+By doing that, you tell mozaic where to look for the source tokens, and where to save the compiled ones. We also use the `localTokensExportPath` inside Postcss to tell it where to look for the SCSS tokens. That way the SCSS will load those ones instead of the ones living in the npm package.
+
+## 2 - Override tokens
+
+To overide tokens, you need to use the **same directory structure and file names as the ones provided in the `@mozaic-ds/tokens`** package. Let's take an example by overriding the Primary brand color.
+
+The default primary color swatch is situated in `@mozaic-ds/tokens/properties/color/base.json`.
+
+Asuming we configured the `mozaic.config.js` as described in the previous chapter, we need to create a `./src/tokens/properties/color/base.json` file.
+
+Note that you can override any value, mozaic will take care of deep merging the source tokens with the one you provide.
+
+For example, in `./src/tokens/properties/color/base.json` :
+
+```Json
+{
+  "color": {
+    "primary-01": {
+      "100": {
+        "value": "#000"
+      },
+    }
+  }
+}
+```
+
+In this example, mozaic will replace the `color.primary-01.100` and set it to black, but all the existing values for `color.primary-01.xxx` will be kept as they where defined in the source.
```

Comment:
and set it to #000 _(black)_

Reply:
```In this example, mozaic will replace the color.primary-01.100 and set it to black, but all the existing values for color.primary-01.xxx will be kept as they where defined in the source.```

Not Clear... will it change the other values based on the new `#000` or my `primary-01-100` will be black but `primary-01-200` will still be green?